Validation fixes

KarolisJasad opened the pull request:
html validation fixes

![image (6)](https://github.com/CERT-Polska/mailgoose/assets/38495063/cfb9d70b-5f34-4698-90d0-390489506f59)
![image (7)](https://github.com/CERT-Polska/mailgoose/assets/38495063/4c30aefd-c62d-4ea5-a47b-219e3303e6cb)

kazet commented on app/templates/check_email.html:
```diff
@@ -53,7 +53,7 @@ <h5 class="card-title">{% trans %}Check configuration by sending an e-mail{% end
                             </div>
 
-                            <img class="spinner waiting" src="/static/images/spinner.svg" />
+                            <img class="spinner waiting" src="/static/images/spinner.svg" alt="spinner-waiting-img"/>
```
As 'alt' will be read by screen reader, would it be possible to add a human-readable message? E.g. "Waiting" - the Polish translation would be "Trwa oczekiwanie".

KarolisJasad replied:
Hi,
So far I can't seem to find a way to add a translation to an "alt" attribute without modifying the {{% trans % }} or making significant changes that would seem out of touch of your current translations and just changing the logic of the translations since I want to keep it the same way as it is. I will look further into this next week. We could just change the alt to "Waiting" and keep it like that as a general message without translation but ofcourse the translation would look prettier and more refined.

kazet replied:
I added something but please check whether the Lithuanian translation is OK, I don't know Lithuanian ;)

kazet commented on app/translations/lt_LT/LC_MESSAGES/messages.po:
```diff
@@ -46,6 +46,10 @@ msgstr ""
 
+#: app/templates/check_email.html:56
+msgid "Waiting"
+msgstr "Laukimas"
```
@KarolisJasad is the Lithuanian translation OK?

KarolisJasad commented:
Changed the translation of lithuanian language for "Waiting" and added the translation to check_running.html img alt aswell.
For some reason the translation like this didn't work for me before, I've tried this too but had to fully reclone/rebuild the project for it to work.
